dbt_invoke/internal/_utils.py

```diff
@@ -132,6 +132,10 @@ def get_project_info(ctx, project_dir=None):
         Path(project_path, macro_path)
         for macro_path in project_yml.get('macro-paths', ['macros'])
     ]
+    # retrieves template information if provided
+    template_path = Path(project_path, 'dbt_invoke_template.yml')
+    if template_path.exists():
+        ctx.config['template_yml'] = parse_yaml(template_path)
```

Comment:
```suggestion
    if template_path.exists():
        ctx.config['template_yml'] = parse_yaml(template_path)
    else:
        ctx.config['template_yml'] = None
```

I am suggesting this because otherwise it seems the key `'template_yml'` may not exist in `ctx.config` if the user does not use a template. This would be a problem when it is called in line 718 of `dbt_invoke/properties.py`

Reply:
Oh, you're right! We could also have used `.get()` in the other file I guess. This works great too, thanks!